crates/block-processor/src/v1/processor.rs

```diff
@@ -129,10 +129,14 @@ where
             return Ok(None);
         }
 
+        let start_time = std::time::Instant::now();
+
         let extractor = Extractor::new(self.constants.clone());
         let shim = ExtractableChainShim::new(chain);
         let outputs = extractor.extract_signet(&shim);
 
+        metrics::record_extraction_time(&start_time);
```

Comment:
we usually directly import these, but given they are metrics and are not relevant to the "main code", do we want to maybe establish this lack of importing as a pattern? wdyt?

Reply:
i don't think it matters